test/thunk_test.ts

```diff
@@ -47,4 +47,41 @@ describe('Thunk', () => {
 
     assert.isTrue(result !== 'Object');
   });
+
+  it('marshaling buffers uses too much memory', async () => {
```

Comment:
maybe add a note that this should be fixed.

Reply:
Done